packages/amplify-graphql-conversation-transformer/src/__tests__/amplify-graphql-conversation-transformer.test.ts

```diff
@@ -130,31 +131,118 @@ describe('ConversationTransformer', () => {
   });
 });
 
-const assertResolverSnapshot = (routeName: string, resources: DeploymentResources) => {
-  const resolverCode = getResolverResource(routeName, resources.rootStack.Resources)['Properties']['Code'];
+const assertAssistantResponseSubscriptionResources = (routeName: string, resources: DeploymentResources) => {
```

Comment:
I can, probably should, and eventually will do these assertions more dynamically based on the pipeline config of the resolver to ensure that function slots added in the future are covered.